tasks/create-github-release/create-github-release.yaml

```diff
@@ -33,19 +37,17 @@ spec:
       description: URL to inspect the created release
   steps:
     - name: create-release-from-binaries
-      image: registry.access.redhat.com/ubi9/ubi:latest@sha256:351ed8b24d440c348486efd99587046e88bb966890a9207a5851d3a34a4dd346
+      image: quay.io/redhat-appstudio/release-service-utils:447ea0580a2cdd48b4091e1df86fab5c3f86d01c
       script: |
-        curl -L \
-          https://github.com/cli/cli/releases/download/v2.32.1/gh_2.32.1_linux_amd64.tar.gz \
-          --output gh_2.32.1_linux_amd64.tar.gz
-        tar -zvxf gh_2.32.1_linux_amd64.tar.gz
-        mv gh_2.32.1_linux_amd64/bin/gh /bin/gh
-        chmod +x /bin/gh
-        cd "$(workspaces.data.path)/$CONTENT_DIRECTORY"
-        gh release create "$RELEASE_VERSION" *.zip *.json *SHA256SUMS --repo "$REPOSITORY"
+        #!/usr/bin/env sh
+        set -ex
+        
+        cd "$(workspaces.data.path)/$(params.subdirectory)/$CONTENT_DIRECTORY"
+        set -o pipefail
+        gh release create v$RELEASE_VERSION *.zip *.json *SHA256SUMS --repo $REPOSITORY | tee $(results.url.path)
+        set +o pipefail
         # We will add .sig file when we have the sigining step is done
-        #gh release create $RELEASE_VERSION *.zip *.json *.sig --repo $REPO
-        echo -n "https://github.com/$REPOSITORY/releases/tag/$RELEASE_VERSION" >  $(results.url.path)
+        #gh release create v$RELEASE_VERSION *.zip *.json *.sig --repo $REPO 
```

Comment:
should be `REPOSITORY` not `REPO` at the end of the line but it doesn't really matter much since it is commented out

Reply:
Good catch! Fixed